voltdb-stored-procedures-maven-quickstart/src/test/resources/projects/basic/goal.txt

```diff
@@ -0,0 +1 @@
+verify
```

Comment:
This tells maven to run "verify" on the generated test project, which runs the integration test.